### DISCUSSION TITLE
平均每周只用三四次，还提示频繁使用？

### DISCUSSION THREAD

**Discussion by jhshilei** [immersive-translate/immersive-translate#1420] [Help]
### 概述 | Summary

youtube的双语字幕功能可以，但网页翻译不行，错误代码429，刚开始是好用的，过了一阵子不行了。字幕一直好用。会是什么原因呢，请高手指点，谢谢！
![1](https://github.com/immersive-translate/immersive-translate/assets/138987831/29fd3bf7-24b1-4156-b249-76301335c3cf)
![2](https://github.com/immersive-translate/immersive-translate/assets/138987831/0d1d9dca-aa70-473f-962d-c5a332dacb4f)

**Comment by keyboard3** [comment]
gemini 的限制，应该是谷歌对gemini有某种程度的限流
